src/server/server.js

```diff
@@ -34,6 +34,7 @@ app.use(helmet());
 app.use(Express.static(__dirname + '/../public', { maxAge: '365d' }));
 
 // routes
+app.use(routes.github);
```

Comment:
I think it would be good to keep it under `/api` path to differentiate between API endpoints and standard pages.